.github/workflows/build-ultraplot.yml

```diff
@@ -53,18 +53,22 @@ jobs:
       - name: Generate baseline from main
         shell: bash -el {0}
         run: |
+          cd $GITHUB_WORKSPACE
           mkdir -p baseline
           micromamba activate ultraplot-dev
           git fetch origin ${{ github.event.pull_request.base.sha }}
           git checkout ${{ github.event.pull_request.base.sha }}
-          pytest --mpl-generate-path=baseline_images
+          pytest --mpl-generate-path=baseline
```

Comment:
Why don't we need the configuration yaml here?

Reply:
Ah good point. Will fix